x-pack/elastic-agent/elastic-agent.yml

```diff
@@ -11,8 +11,11 @@ outputs:
   default:
     type: elasticsearch
     hosts: [127.0.0.1:9200]
-    username: elastic
-    password: changeme
+    api_key: "example-key"
+    # Note that basic auth is deprecated and will be removed in 8.0
```

Comment:
This does not apply to the general output. +1 on having the api_key here but username / password for the output part stay around AFAIK.

Reply:
ah, much simpler then. i'll just log it there.